docs_devel/docs/40.ContributingDocument.md

```diff
@@ -1,27 +1,24 @@
-# Contributing OmegaT documentation
+# Contributing to OmegaT documentation
 
-OmegaT Manuals and User guides are co-developed on OmegaT program repository.
+The OmegaT related documentation is co-developed in the OmegaT repository.
 
-You can see a document source in`doc_src/` folder in OmegaT project source code repository.
+The source files are in `doc_src/`. Documentation source uses the [Docbook 4.5](https://tdg.docbook.org/tdg/4.5/docbook.html) format.
 
 ## Documentation rules and procedures
 
-Please contact to [Documentation manager](https://omegat.org/about) at [omegat-development mailing list]
-(https://sourceforge.net/projects/omegat/lists/omegat-development) for details.
+Documentation contribution follows the same rules are code contributions. See [Contributing to OmegaT development](01.ContributingOmegaT.md) for more information.
 
-## manual snapshot release
+You can also consult our SourceForge [documentation tracker](https://sourceforge.net/p/omegat/documentation/) for potential contributions.
 
-When developer update manuals in source code, CI/CD automatically build HTML manuals and published
-on web server. You can see snapshot version of HTML manual at https://omegat.sourceforge.io/manual-snapshot/
+## Manual snapshot release
+
+When the document source is modified, the CI/CD processes automatically build the HTML manuals and publish that snapshot
+on the web server at [https://omegat.sourceforge.io/manual-snapshot/](https://omegat.sourceforge.io/manual-snapshot/).
 
 ### License
 
-All the files in `doc_src` folder and below are under the terms of the GNU General
+All the files in `doc_src` folder and below are published under the terms of the GNU General
```

Comment:
@brandelune 
I'm not a licence expert, but do software licenses apply to documentation?
Should this be one of the Creative Commons licenses?

Reply:
The GPL is not specifically a software licence. It is a source code distribution licence. Our docbook sources are not supposed to have issues that can only be solved by using documentation specific licences. But if you find such issues, we need to investigate how to fix them, because changing the licence would be a major pain.

Reply:
If XML is considered code, then there's probably no problem because the docbook sources can be defined as "source code".

I'll trust that whoever first chose the licence checked this at the time! :)

Reply:
Originally, the manual was in HTML. There was no build process involved. The whole set was distributed under the GPL, and it just stayed like that. The Emacs manuals use the GFDL 1.3+ which makes sense, if I remember well, because they can also be distributed as books (cf. invariant clauses, etc.)

Reply:
Interesting.

I found [this interesting thread](https://opensource.stackexchange.com/questions/741/with-gpl-what-does-source-code-mean-for-non-software) that's relevant.

If I understand it correctly, GPL for the docbook sources is fine. The actual manual(s) produced from those sources should probably be under the GFDL or one of the Creative Commons licences.

Reply:
What practical difference would that make? Or rather, what problem is that a solution for?

Reply:
If I understood the thread (and GNU GPL FAQ) correctly, licensing the actual manual (output HTML, PDF, etc.) under the GPL would require including the contents of `doc_src` with the manual itself, which I think would be (a) cumbersome for us, and (b) tedious for users.

Of course, I might simply be misunderstanding the answers in the thread and FAQ.

Reply:
FSF recommend GFDL for document rather than GPL in several reason.
https://www.gnu.org/licenses/license-recommendations.html

It is something extra-ordinal to apply GPL to a large manual document like OmegaT manual, so I try to add explanation here.

Reply:
I understand that our choice of licence may not be the best, but we currently do not have issues that need to be solved by a change of licence.

It could be a fun exercise to track down the various authors that share the copyright on the current document though.

Reply:
Well, the statement in the developer documentation here only applies to the docbook sources, so as far as those go, I think the developer documentation is fine as is in terms of contents.

The broader licensing issue may (or may not) be something to look at further down the road in the larger context of the entire project.
Have we explicitly applied the GPL (or any other licence) to the products output from those sources?

Reply:
The GPL applies to the deliverables (the "program") and forces the developers to provide sources on demand so that users can modify the deliverable. The GPL is not a source file distribution licence.